test/hotspot/jtreg/compiler/c2/Test7046096.java

```diff
@@ -30,6 +30,18 @@
  *    compiler.c2.Test7046096
  */
 
+
+/*
+ * @test id=stringConcatInline
+ * @bug 7046096
```

Comment:
```suggestion
 * @bug 7046096 8357822
```
I'd at the new number here. But probably optional.